### DISCUSSION TITLE
flux-accounting system instance feedback

### DISCUSSION THREAD

**Discussion by cmoussa1** [flux-framework/flux-accounting#156] [General]
I figure this discussion thread could serve as a place to leave any detailed feedback or suggestions as a result of testing flux-accounting as an installed component of a Flux system instance. Any feedback or suggestions that require fixes or additional features can be created into issues so they can be tracked. I can write some immediate thoughts and edit this if new things come to mind as we are testing it.

#### installing/configuring flux-accounting

- were there any hiccups installing flux-accounting alongside flux-core?
- how was it creating the flux-accounting DB and configure the cron job scripts and functions that re-calculate job usage values and fairshare values? (`./flux_update_fshare`, `flux account update-usage`)
- how was it loading the multi-factor job priority plugin and setting up the cron job that sends payload data from the flux-accounting DB to the plugin (`flux python bulk_update.py`)

#### using the accounting commands

- did any of the subcommands to handle user or bank accounts behave unexpectedly? were there subcommands missing? 

#### fetching job records

- was the format of the job record(s) output satisfactory? was there any additional information required when fetching job records?

This list of items and points of feedback is dynamic, and I expect it to change as we start to use flux-accounting more and in different ways. I will continue to update it in the future, and convert points of feedback into issues as they come up.

**Comment by grondo** [comment]
If there is some online documentation for setting up flux-accounting, it may be useful to link that here.

If not, maybe adding a section for flux-accounting to the [Flux Administrator's Guide](https://flux-framework.readthedocs.io/en/latest/adminguide.html) would be a good idea at this point.

**Reply by cmoussa1** [reply]
Yes, absolutely. That is actually what I was thinking of working on next! I have #148 open currently for updating the docs here for flux-accounting (it's been a while since I have), but also updating the Flux Administrator's Guide sounds like a great idea too.
